src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -406,6 +408,7 @@ particular backend, run "pachctl deploy storage <backend>"`,
 	deploy.PersistentFlags().StringVar(&imagePullSecret, "image-pull-secret", "", "A secret in Kubernetes that's needed to pull from your private registry.")
 	deploy.PersistentFlags().StringVar(&dashImage, "dash-image", "", "Image URL for pachyderm dashboard")
 	deploy.PersistentFlags().BoolVar(&noGuaranteed, "no-guaranteed", false, "Don't use guaranteed QoS for etcd and pachd deployments. Turning this on (turning guaranteed QoS off) can lead to more stable local clusters (such as a on Minikube), it should normally be used for production clusters.")
+	deploy.PersistentFlags().BoolVar(&noRBAC, "no-rbac", false, "Don't deploy RBAC roles for Pachyderm.")
```

Comment:
Similar to above, would `BoolVar(&rbac, "rbac", true, ...)` be clearer than `BoolVar(&noRBAC, "no-rbac", false, ...)`?

Reply:
Same here, I do think `--rbac` is clearer... but it means that the default is no rbac, which seems like a worse default to me.

Reply:
I know that booleans in general default to false, but since you can set the flag default to true, and you set the option directly from the flag either way, I think you can just flip both (and change `false` to `true` on this line) and it'll work. But like I said, this is all pretty cosmetic, so it's up to you

Reply:
Ah, I see, and then you could do `--rbac=false` to explicitly set it to false. I can definitely see the argument for that, since it's weird having the `no-` prefix. On the other hand it also seems weird that you'd have a flag `--rbac` where just passing it does nothing since it's already the default value. I could see arguments either way but given that we already have `no-guaranteed` I think the weirdest thing would be to change one and not the other, and since `no-guaranteed` is already out in the wild I'm reluctant to change it around. I think I'm just going to stick with what I've got for now.